interfaces/Portalicious/tsconfig.json

```diff
@@ -3,16 +3,22 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
-    "baseUrl": "./src",
+    "baseUrl": "./",
     "paths": {
       "@121-service/*": ["./../../services/121-service/*"],
-      "~environment": ["environments/environment"],
-      "~/*": ["app/*"]
+      "~environment": ["./src/environments/environment"],
+      "~/*": ["./src/app/*"]
     },
     "outDir": "./dist/out-tsc",
     "strict": true,
+    // TODO: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/28804
+    "strictPropertyInitialization": false,
+    // TODO: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/28419
+    "noImplicitAny": false,
+    // TODO: https://dev.azure.com/redcrossnl/121%20Platform/_workitems/edit/27718
+    "useUnknownInCatchVariables": false,
```

Comment:
I am forced to do this because otherwise, the imports from the 121-service don't work, and the compiler complains. That's because these flags are set to false in the 121-service, so there is lots of code that doesn't comply with these rules.

For things like explicitly using `any`, we have eslint keeping us in check. I would still rather not do this. I spent an hour trying to figure out how to avoid it to no avail. Maybe it's worth some pair programming.
 
Having to make this change makes me very sad.